scss/_reboot.scss

```diff
@@ -27,6 +27,10 @@
 
 :root {
   font-size: $font-size-root;
+
+  @media (prefers-reduced-motion: no-preference) {
+    scroll-behavior: smooth;
```

Comment:
Maybe we should make this configurable (Sass variable)?

Reply:
Neat idea; I introduced `$enable-smooth-scroll` (`true` as a default) to ease override.